### PR TITLE
It's the final countdown!

### PR DESCRIPTION
**Pull request by lwratten**
## PR Description
Okay quick rundown of changes:
- implemented string querying
- used tabulate to format `--overview` output
- added `--pretty` option to use tabulate to print out db query results
- check `-o` directory exists
- made a change to README formatting for installation

## PR checklist
 - [X] This comment contains a description of changes (with reason)
 - [X] If you've fixed a bug or added code that should be tested, add tests!
    Some quick things to test:
`falcon_multiqc query -tm verifybamid AVG_DP '>' 24.5 -tm fastqc "kmer_content" '==' "fail" --pretty`
`falcon_multiqc query -tm verifybamid AVG_DP '>' 24.5 -tm fastqc "kmer_content" '==' "fail"`
`falcon_multiqc query --overview`

 - [X] `README.md` is updated - will do ASAP!

### CODE REVIEW

**Review by Kento-Bento — APPROVED** [review]
Looking good!

File: README.md [hunk 1]
```diff
@@ -2,12 +2,12 @@
...
-3. Ensure your system has the tools in requirements.system installed.
-4. python3 -m venv env create a python virtual environment if you haven't already
-5. source env/bin/activate go into the virtual environment
-6. pip install . or pip install --editable . if developing
+1. Clone the repository to your local machine using `git clone https://github.com/lwratten/project-falcon.git`.
+2. Navigate to the repository using `cd project-falcon`.
+3. Ensure your system has the tools in `requirements.system` installed.
+4. Use `python3 -m venv env` to create a python virtual environment if you haven't already.
+5. Use `source env/bin/activate` to go into the virtual environment.
+6. Install software requirements using `pip install .` or `pip install --editable .` if developing.
```
Comment (Kento-Bento) [inline README.md:10]: Actually, we discovered the `pip install .` might be slightly bugged and not work, only pip install -e . seems to work, so maybe change this?

Comment (lwratten) [reply README.md:10]: I think we might have to make quite a few changes to the docs anyway so maybe i'll make a docs pr today after I've gone through it! 👍

Comment (Celinet21) [reply README.md:10]: sweet thx for that

**Review by Celinet21 — APPROVED** [review]
cool stuff

**Pull request merged**